__tests__/app.test.js

```diff
@@ -66,6 +66,84 @@ describe("GET: /api/articles/:article_id", () => {
                 expect(response.body.msg).toBe("Article does not exist");
             });
     });
+    test('PATCH: 200 update votes on an article', () => {
+        const votes = {inc_votes: 10}
+        return request(app)
+        .patch("/api/articles/1")
+        .send(votes)
+        .expect(200)
+        .then((response) => {
+            const { article } = response.body
+            expect(article).toMatchObject({
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: "2020-07-09T20:11:00.000Z",
+                article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                votes: 110 
+            });
+        })
+    });
+    test('PATCH: 200 ignores extra properties', () => {
+        const votes = {inc_votes: 10, isPointsCool: true}
+        return request(app)
+        .patch("/api/articles/1")
+        .send(votes)
+        .expect(200)
+        .then((response) => {
+            const { article } = response.body
+            expect(article).toMatchObject({
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: "2020-07-09T20:11:00.000Z",
+                article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                votes: 110 
+            });
+        })
+    });
+    test("PATCH: 404 sends an appropriate status and error message when provided with a non-existent but valid id", () => {
+        const votes = {inc_votes: 10}
+        return request(app)
+        .patch("/api/articles/999")
+        .send(votes)
+        .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe("Article does not exist");
+            });
+    });
+    test("PATCH:400 sends an appropriate status and error message when provided with a invalid id", () => {
+        const votes = {inc_votes: 10}
+        return request(app)
+        .patch("/api/articles/abc")
+        .send(votes)
+        .expect(400)
+        .then((response) => {
+            expect(response.body.msg).toBe("Bad Request - type conversion issue");
+            });
+    });
+    test("PATCH: 400 lacking votes", () => {
```

Comment:
Ideally this would be Status 200, if inc_votes is missing, should not update article and return original single article object (this may be included as a 400 but that could make for a brittle test suite)